lib/beaker-answers/versions/version34.rb

```diff
@@ -30,15 +29,15 @@ def generate_answers
       rbac_database_password       = "'#{answer_for(@options, :q_rbac_database_password)}'"
 
       console_services_hash = {
-        :q_classifier_database_user     => classifier_database_user,
-        :q_classifier_database_name     => classifier_database_name,
+        :q_classifier_database_user => classifier_database_user,
```

Comment:
Let's turn off whatever is fiddling with alignment here.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutalignhash disabled